scripts/test_env.py

```diff
@@ -50,7 +51,10 @@ def __init__(self, args: argparse.Namespace):
             peer_entry = {"address": f"{peer.host_ip}:{peer.p2p_port}", "public_key": peer.public_key}
             self.trusted_peers.append(json.dumps(peer_entry))
         os.environ["SUMERAGI_TRUSTED_PEERS"] = f"[{','.join(self.trusted_peers)}]"
-
+        
+        if is_startup_benchmark:
+            del self.peers[1:]
```

Comment:
Can't you just use `1` instead of `args.n_peers` in line 41?

Reply:
I would like to, but iroha refuses to start unless there are enough other peers in the network_topology. So I let it setup the full network topology and then this stops all but one node from running.